### DISCUSSION TITLE
Profiles' local location on Windows?

### DISCUSSION THREAD

**Discussion by anseltaft** [Eugeny/tabby#8184] [General]
Hi y'all,

I wondered if someone could please tell me where profiles are stored locally if I'm using Tabby in Windows? I'd like to back them up from time to time and, should I move to a new computer, know where to go.

Cheers!

**Comment by Eugeny** [comment]
In the config file - Settings -> Config file -> Show config file

**Reply by anseltaft** [reply]
I found it, thank you.

I also came across the local file I can move to a new computer as needed:
C:\Users\me\AppData\Roaming\tabby\config.yaml

In addition I set up config sync tonight so all's good. Cheers
